bots/task/testmap.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
One of these days I'll send a PR that removes all these lines :-)

Reply:
eeehm :D I just copied the whole `tests-scan` and then dropped everything after the map. Removing it would be awesome :) I may send some PR, it should be one `sed` command